### PR TITLE
SCMOD-4243:  Fixed timeouts and healthchecks

### PR DESCRIPTION
**Pull request by rado-straka**
https://jira.autonomy.com/browse/SCMOD-4243
http://sou-jenkins2.hpeswlab.net/job/Autoscaler/job/Autoscaler%7Eautoscaler%7ESCMOD-4243%7ECI/1/

### CODE REVIEW

File: autoscale-marathon-container/pom.xml [hunk 1]
```diff
@@ -152,9 +152,6 @@
                                         <args>/maven/scaler.sh</args>
                                     </exec>
                                 </entryPoint>
-                                <healthCheck>
-                                    <cmd>curl --fail http://localhost:8081/healthcheck || exit 1</cmd>
-                                </healthCheck>
```
Comment (dermot-hardy) [inline autoscale-marathon-container/pom.xml]: Why are you removing the Docker healthcheck?

File: autoscale-dockerswarm-container/pom.xml [hunk 2]
```diff
@@ -125,9 +125,6 @@
                                         <args>/maven/scaler.sh</args>
                                     </exec>
                                 </entryPoint>
-                                <healthCheck>
-                                    <cmd>curl --fail http://localhost:8081/healthcheck || exit 1</cmd>
-                                </healthCheck>
```
Comment (dermot-hardy) [inline autoscale-dockerswarm-container/pom.xml]: Why are you removing the Docker healthcheck?

File: autoscale-dockerswarm-container/pom.xml [hunk 3]
```diff
@@ -158,18 +155,24 @@
...
-                                    <healthy>true></healthy>
+<!--                                <wait>
+                                    <http>
+                                        <url>http://${docker.host.address}:${scaler.port}/healthcheck</url>
+                                        <status>200</status>
+                                    </http>
                                     <time>45000</time>
                                     <shutdown>500</shutdown>
-                                </wait>
+                                </wait>-->
```
Comment (dermot-hardy) [inline autoscale-dockerswarm-container/pom.xml]: You're commenting out the wait altogether?

File: autoscale-dockerswarm-container/pom.xml [hunk 4]
```diff
@@ -158,18 +155,24 @@
                                 </assembly>
                             </build>
                             <run>
+<!--                                <ports>
+                                    <port>${scaler.port}:8081</port>
+                                </ports>-->
```
Comment (dermot-hardy) [inline autoscale-dockerswarm-container/pom.xml]: If the ports aren't required then remove them rather than commenting them out.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Comment by rado-straka** [comment]
Updated.

Interesting to mention is that with these healthchecks build time takes 2 minutes more now. For both containers healthcheck time moved from 6 seconds to 30 seconds.
http://sou-jenkins2.hpeswlab.net/job/Autoscaler/job/Autoscaler~autoscaler~SCMOD-4243~CI/

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**